src/Normalizer/Relation/EmbedOneFieldNormalizer.php

```diff
@@ -49,6 +50,12 @@ public function normalizeField(
             return null;
         }
 
+        if (true === class_exists('Doctrine\Common\Persistence\Proxy')
```

Comment:
@TiMESPLiNTER please do not use true === if the return value of a method is a boolean